scripts/adage_pancancer.py

```diff
@@ -33,10 +33,39 @@
 import numpy as np
 import pandas as pd
 
+from keras.engine.topology import Layer
 from keras.layers import Input, Dense, Dropout, Activation
-from keras.models import Model
-from keras import optimizers
+from keras.models import Sequential, Model
+import keras.backend as K
 from keras.regularizers import l1
+from keras import optimizers, activations
+
+
+class TiedWeightsDecoder(Layer):
+    """
+    Transpose the encoder weights to apply decoding of compressed latent space
+    """
+    def __init__(self, output_dim, encoder, activation=None, **kwargs):
+        self.output_dim = output_dim
+        self.encoder = encoder
+        self.activation = activations.get(activation)
```

Comment:
Is this suppose to be activation function? I'm a lil lost on the purpose of this line.

Reply:
it turns an input activation string (say, for example, `'relu'`) and converts it to an object Keras can deal with - see https://github.com/keras-team/keras/blob/master/keras/activations.py#L102